tasties_app/templates/tasties_app/single_recipe.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'tasties_app/base.html' %}
+{% load static %}
+{% block css %}
+    {% load bootstrap_icons %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/single_recipe.css' %}" />
+{% endblock %}
+{% block content %}
+    <div class="recipe-body">
+        <div class="recipe-header">
+            <h1>{{ recipe.title }}</h1>
+            <h2>{{ recipe.description }}</h2>
+            <div class="recipe-header-info">
+                <span class="author-name">by @{{ recipe.author_id }}</span>
+                <span class="recipe-date">{{ recipe.publication_date }}</span>
+            </div>
+        </div>
+
+        <div class="categories">
+            {% for category in categories %}
+                <button class="btn btn-light" name="{{ category.category_name }}">{{ category.category_name }}</button>
```

Comment:
I recommend a slightly darker/different color for the category tags, so they stand out more against the background.